hub/www/item_listing.py

```diff
@@ -7,10 +7,15 @@ def get_context(context):
     fields = ['published', 'route', 'image', 'name', 'company_name', 'price', 'stock_qty', 'currency', '`tabHub Item Review`.content', 'count(`tabHub Item Review`.content) as reviews_count']
     group_by = 'name'
     filters = {'published': 1}
-    page_number = int(frappe.local.request.args.get('page_number', 1))
-    paginator = Paginator('Hub Item', page_number=page_number, fields=fields, filters=filters, order_by='name', group_by=group_by)
+    page_number = int(frappe.local.form_dict.get('page_number', 1))
+    search = frappe.local.form_dict.search
+    or_filters = None
+    if search:
+        or_filters = [
```

Comment:
So we have just added a `keywords` field in item, that concatenates all the important field values in plain text (not yet set its db column to FULLTEXT though). So it can be used to run the `like` query.